src/Cassandra/Mapping/CqlBatch.cs

```diff
@@ -29,14 +29,14 @@ public CqlBatch(MapperFactory mapperFactory, CqlGenerator cqlGenerator)
             _statements = new List<Cql>();
         }
 
-        public void Insert<T>(T poco, CqlQueryOptions queryOptions = null)
+        public void Insert<T>(T poco, CqlQueryOptions queryOptions = null, int? ttl = null)
```

Comment:
Creating a new parameter (even if its with a default value) is a binary break, we must expose it as an overload if we want to deliver it in a minor / patch version: http://stackoverflow.com/questions/1456785/a-definitive-guide-to-api-breaking-changes-in-net